entries/k/kit.edu.json

```
@@ -6,6 +6,9 @@
       "totp",
       "custom-hardware"
     ],
+    "custom-hardware": [
+      "Yubico OTP"
```

```suggestion
      "Yubico OTP",
      "KIT Token"
```

Source: [KIT Token](https://www.scc.kit.edu/dienste/11223.php)